benchmark/shell_env.sh

```diff
@@ -12,7 +12,7 @@ export BENCHMARK_DATASET=${BENCHMARK_DATASET:-hits}
 export BENCHMARK_TYPE=${BENCHMARK_TYPE:-"pr"}
 
 # hits
-export HITS_DATA_URL=${HITS_DATA_URL:-"http://127.0.0.1:8902/data.gz"}
+export HITS_DATA_URL=${HITS_DATA_URL:-"/data/hits.parquet"}
```

Comment:
Is this variable still useful?

Reply:
Yes, I put the data in this location on the gitlab node

Reply:
I don't see where this variable is used, could you show me

Reply:
I indeed haven't used it, I'll delete it.